clutter/SplitScreenScanner
# UI Updates

## Pivotal Ticket
https://www.pivotaltracker.com/story/show/156866606

## Feature Images
Updated UI elements based on QA feedback to help the SplitScreenScanner conform better to the style guide.
![img_0065 2](https://user-images.githubusercontent.com/10718632/39216003-e6a2e0a4-47ce-11e8-9485-0f949c5a7b10.PNG)

![img_0070](https://user-images.githubusercontent.com/10718632/39216089-32b58a32-47cf-11e8-8a12-07cf62fc3923.PNG)

![img_0067](https://user-images.githubusercontent.com/10718632/39216020-f7f5f81e-47ce-11e8-9ecb-4cb3b42c099f.PNG)

![img_0068](https://user-images.githubusercontent.com/10718632/39216025-fba51a12-47ce-11e8-8d99-575fdbf712c3.PNG)

![img_0069](https://user-images.githubusercontent.com/10718632/39216036-ff4bbe46-47ce-11e8-9354-0b8dfb507e89.PNG)

## What To Test?
- Open the scanner and scan the wrong starting barcode, making sure the wrong starting barcode scanned error appears
- Scan the correct starting barcode and make sure the updated "Scan an item to start loading" cell apears
- Make a few scans and make sure the updated item history cell appears after each scan
- Wait for the scanning session to expire and make sure the updated "Scan to Continue" screen appears
